canvas/text.go

```diff
@@ -21,6 +21,11 @@ type Text struct {
 	Text      string         // The string content of this Text
 	TextSize  float32        // Size of the text - if the Canvas scale is 1.0 this will be equivalent to point size
 	TextStyle fyne.TextStyle // The style of the text content
+
+	// FontSource defines a resource that can be used instead of the theme for looking up the font.
```

Comment:
We may want to think about compatability with the future where we may expose a public API to retrieve system fonts. This could probably work with that if that API were a way to obtain `fyne.Resource`s by looking up on the OS by font family name. Just something to think about

Reply:
Actually a FontFamilyName might be a better API for that - so the developer doesn't have to think about using different FontSource resources for different text styles?

Reply:
As discussed on Slack the lookups etc could be added but family etc is just metadata that operates at a higher level than this simple configuration of a canvas item metadata.